core/src/databases/database.rs

```diff
@@ -0,0 +1,108 @@
+use serde::Serialize;
+use serde_json::Value;
+
+#[derive(Debug, Serialize)]
+pub struct Database {
+    created: u64,
+    data_source_id: String,
+    database_id: String,
+    name: String,
+}
+
+impl Database {
+    pub fn new(created: u64, data_source_id: &str, database_id: &str, name: &str) -> Self {
```

Comment:
I don't see how you're going to use this since you'll want to create the database in store to get a created time no ?

Reply:
The `store` module is calling this function to return objects after pulling them from the DB (so the created already exists)

The input to the store functions is just the individual fields not an object